test/python/test_pack.py

```diff
@@ -160,7 +150,7 @@ def rpm_archive_with_custom_units(tmpdir, light_project):
 # #####
 # Tests
 # #####
-def test_tgz_pack(tgz_archive, tmpdir):
```

Comment:
Seems you try to save several bytes of source code)

I don't see significant reasons to rename. Yes it will be displayed as "test_pack::test_tgz" but it's a strange perfectionism

Reply:
No, I just think, that `test_pack::test_tgz` looks better than `test_pack::test_pack_tgz` ;(